docs/js/redux-injectors.md

```diff
@@ -0,0 +1,27 @@
+# `redux-injectors`
+
+`redux-injectors` is a library we made ourselves by moving code we used to have
+in the utils folder into a seperate library. It allows you to dynamically load
```

Comment:
I would replace the first sentence with something like this:

> [`redux-injectors`](https://github.com/react-boilerplate/redux-injectors) is an official `react-boilerplate` companion library. We built it so that it can be used and maintained independently from `react-boilerplate`.
> 
> It allows you to dynamically load...

It feels slightly more professional IMO.